internal/apps/assignment/projector.go

```diff
@@ -169,6 +169,11 @@ func PollSlide() projector.CallableFunc {
 		pollData["options"] = b
 
 		if pollState == poll.StatePublished {
+			pollData["amount_global_yes"] = []byte("null")
+			if !bytes.Equal(pollInfo["amount_global_yes"], []byte("0")) {
+				pollData["amount_global_yes"] = pollInfo["amount_global_yes"]
+			}
+
```

Comment:
This seems unnecessary complicated. You get the same result if you remove lines 172, 173 and 175.

If `pollInfo["amount_global_yes"]` is `null` (the else case) then you can also write it into the variable `pollData["amount_global_yes"]`. This means that the if case and the else case are the same.